docs/Dialects/FIRRTL/FIRRTLAnnotations.md

```diff
@@ -299,6 +299,31 @@ Example:
 }
 ```
 
+### Convention
+
+| Property   | Type   | Description                 |
+| ---------- | ------ | --------------------------- |
+| class      | string | `firrtl.linkage.Convention` |
```

Comment:
do we want this to be the level of the annotation, or do we want something more blanket `firrtl.annotations.PublicModule`?

which will act as a proxy for a `public module` that might get added to firrtl spec?